Enterprise v4 (#156)
astronomer/cloud-docs: cloud/upgrade-runtime.md

sidebar_label: 'Upgrade Runtime'
title: 'Upgrade Astronomer Runtime'
id: upgrade-runtime

> description: Upgrade Astronomer Runtime in an Astronomer project.

Reviewer: ```suggestion
description: Upgrade your Deployment's version of Astronomer Runtime.
```
